sectorstatemgr/pubsub.go

```diff
@@ -0,0 +1,60 @@
+package sectorstatemgr
+
+import "sync"
+
+type PubSub struct {
```

Comment:
Is it possible to use an existing pubsub library rather than creating a new one?
In other places in the code we use "github.com/hannahhoward/go-pubsub" and EventEmitter from "github.com/libp2p/go-libp2p/core/event"

Reply:
I don't see references to `EventEmitter` in Boost. `github.com/hannahhoward/go-pubsub` is certainly going to introduce more complexity and indirection if we decide to use it.